src/Views/SimulatorLog/SimulatorLog.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import "./SimulatorLog.css";
+import { useSelector } from "react-redux";
+import { getCurrentExecution, IExecutionLogState } from "./simulatorLogSlice";
+
+/**
+ * Component for logging the commands of the simulator.
+ */
+export const SimulatorLog: FunctionComponent = () => {
+  const [tableData, setTableData] = useState({} as IExecutionLogState);
+  const currentExecution = useSelector(getCurrentExecution());
+
+  useEffect(() => {
+    setTableData(currentExecution);
+  }, [currentExecution]);
+
+  return (
+    <table>
+      <thead
+        style={{
+          display: "table",
+          tableLayout: "fixed",
+        }}
+      >
+        <tr>
+          <th>Timestamp</th>
+          <th>Command</th>
+        </tr>
+      </thead>
+      <tbody
+        style={{
+          display: "block",
```

Comment:
can you move this to the `SimulatorLog.css` file? Also, I don't think you should be overriding the display of the tbody unless you had a specific reason for it?

Reply:
moved to css but whenever I remove display the UI looks weird